### PR TITLE
Camera error handling

### PR DESCRIPTION
**Pull request by adamweeks**

### CODE REVIEW

File: packages/node_modules/@ciscospark/widget-meet/src/messages.js [hunk 1]
```diff
@@ -22,6 +22,22 @@ export default defineMessages({
...
+    id: `ciscospark.container.meet.button.ok`,
+    defaultMessage: `Ok`
+  },
+  callErrorDeviceNotFound: {
+    id: `ciscospark.container.meet.message.deviceNotFound`,
+    defaultMessage: `No calling devices found`
+  },
+  callErrorNotAllowed: {
+    id: `ciscospark.container.meet.message.deviceNotFound`,
+    defaultMessage: `Media access has been blocked by browser`
```
Comment (bzang) [inline packages/node_modules/@ciscospark/widget-meet/src/messages.js]: Can we change this to: `Cisco Spark cannot access your camera. Please check your browser settings and try again."

File: packages/node_modules/@ciscospark/widget-meet/src/messages.js [hunk 2]
```diff
@@ -22,6 +22,22 @@ export default defineMessages({
     id: `ciscospark.container.meet.button.hangup`,
     defaultMessage: `Hangup`
   },
+  okButtonLabel: {
+    id: `ciscospark.container.meet.button.ok`,
+    defaultMessage: `Ok`
+  },
+  callErrorDeviceNotFound: {
+    id: `ciscospark.container.meet.message.deviceNotFound`,
+    defaultMessage: `No calling devices found`
```
Comment (bzang) [inline packages/node_modules/@ciscospark/widget-meet/src/messages.js]: Can we change this to: "Cisco Spark cannot find your camera. Please check your browser settings, and that your camera is accessible, and try again."

File: packages/node_modules/@ciscospark/widget-meet/src/messages.js [hunk 3]
```diff
@@ -34,6 +50,10 @@ export default defineMessages({
     id: `ciscospark.container.meet.message.noWebRTCBrowserSupport`,
     defaultMessage: `Video calling is not yet supported in your browser.`
   },
+  unableToCallMessage: {
+    id: `ciscospark.container.meet.message.unableToCallMessage`,
+    defaultMessage: `Unable to connect call`
```
Comment (bzang) [inline packages/node_modules/@ciscospark/widget-meet/src/messages.js]: Cisco Spark cannot connect your call. Please try again.

Comment (adamweeks) [inline packages/node_modules/@ciscospark/widget-meet/src/messages.js]: Unable to call is the main error message while the others are the subtext. Probably don't need "Cisco Spark" in both.

Comment (adamweeks) [inline packages/node_modules/@ciscospark/widget-meet/src/messages.js]: Actually, let me just make them one message without subtext

**Pull request merged**